share/completions/wg-quick.fish

```diff
@@ -0,0 +1,18 @@
+set -l valid_subcmds up down strip save
+
+function __wg_complete_interfaces
+    wg show 2>| string replace -rf 'Unable to access interface (.*): Operation not permitted' '$1'
```

Comment:
Ooof, this seems... awkward. Is there really no better way than to parse an error message?

If you get an error message, is the argument actually useful?

Reply:
`wg-quick` looks for interfaces in `/etc/wireguard/`, which has `700` permissions so we cannot get completion candidates from there. This was the only way I could find that works - however I guess will need to prefix with `LC_ALL=C` here. `wg show` always shows the name of the currently connected interface in its error message, so it is perfect as the completion in this case.

Reply:
>wg-quick looks for interfaces in /etc/wireguard/, which has 700 permissions so we cannot get completion candidates from there.

Okay, wait... what happens if you are the owner (I'm assuming root?)?

What does `wg show` say then?

Reply:
Ping @ammgws: Does this work if you're root? I'm assuming it doesn't error out then, but do the completions still work?

Reply:
@faho Sorry for the late reply. It doesn't error out, just gives garbage candidates for the completion:
```sh
>root@mydesktop ~# wg-quick down [TAB]
  allowed ips: 0.0.0.0/0, ::/0        private key: (hidden)                                   
  endpoint: xxx:51820                 public key: xxxx
  fwmark: 0xca6c                      transfer: 414.65 KiB received, 141.07 KiB sent          
  latest handshake: 50 seconds ago    interface: <completion target>                                     
  listening port: 49937               peer: xxxx
 ```

Reply:
Actually it looks like that works for non-root too, so we can just get rid of the kluge altogether:

```sh
>wg show
Unable to access interface mlvd-ch2: Operation not permitted
[1]>wg show interfaces
my_wg_interface
```